HW04.Rmd

```diff
@@ -78,42 +151,114 @@ Assume that:
 ### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
 
 ```{r race}
-
+#select data and count
+race_intent <- gun_deaths %>% 
+    select(intent, race) %>% 
+    filter(intent %in% c("Suicide", "Homicide"), race %in% c("White", "Black", "Hispanic")) %>% 
+    group_by(race, intent) %>% 
+    count() %>% 
+    ungroup() %>% 
+    group_by(race) %>% 
+    mutate(total_race = sum(n)) %>% 
+    mutate(percentage = round(n / total_race * 100, digits = 2)) %>% 
+    select(-total_race)
 ```
 
 (hint maybe looking at percentages would be good)
 ```{r race_2}
-
+#plot the data and set position to "fill" to show ratio
+ggplot(race_intent, aes(x = race, y = n, fill = intent)) +
+    geom_col(position = "fill") +

```

Comment:
I set position = "dodge" to have the bars next to each other. But leaving it this way seems to be better for visualizing differences in proportions between groups. I'll keep that in mind!